opennms-base-assembly/src/main/filtered/etc/snmp-graph.properties.d/netsnmp-graph.properties

```diff
@@ -228,6 +228,87 @@ report.netsnmp.cpuStatsFull.command=--title="CPU Statistics" \
  COMMENT:" " \
  LINE1:float15#cc0000:"15 minutes\\n"
 
+report.netsnmp.cpuStatsFuller.name=CPU Statistics
```

Comment:
The report `netsnmp.cpuStatsFuller` is missing on top of the file in the report section.

Reply:
fixed